_posts/2021-05-02-release-1.x.x.md

```diff
@@ -0,0 +1,112 @@
+
+## GSoC? 
+
+## Notable new additions
+
+- FHI-AIMS reader? NAMD binary coordinate format? ITPParser?
+- ParmEd and Chemfiles converters?
+- PCA changes?
+- AnalysisBase refactors? (DensityAnalysis, etc..)?
+- Improved selections?
+- New hbond code? (w/ correlation module)
+- get_connections (PR #3160)
+
+## New support for 32 bit Windows?
+
+- Add some blurb here
+
+## Notable improvements
+
+- Element parsing in PDB and XYZ files?
+- TPR parser -> GMX 2020?
+- New contacts radius_cut_q method?
+- tqdm?
+
+## Miscellaneous performance improvements
+
+- Ramachandran? (PR #2706)
+- TPR file reading? (PR #2804)
+
+## Deprecations
+
+This release brings severals deprecations as the package heads towards version 2.0.0.
+
+The following parts of the analysis code will be removed/changed in version 2.0.0:
+- analysis.hole is deprecated in favor of analysis.hole2.
+- analysis.hbonds.HydrogenBondsAnalysis is deprecated in favor of analysis.hydrogenbonds.hbond_analysis.
+- analysis.density.density_from_Universe() is deprecated in favor of analysis.density.DensityAnalysis.
+- The notwithin_coordinates_factory() and density_from_PDB() methods of analysis.density are deprecated.
+- analysis.waterdynamics.HydrogenBondLifetimes is deprecated in favor of analysis.hydrogenbonds.hbond_analysis.HydrogenBondAnalysis.lifetime() (to be implemented in version 2.0.0)
+- analysis.leaflets.LeafletFinder() will no longer accept a filename, in 2.0.0 only Universes will be supported as inputs.
+- analysis.helanal is deprecated and will be replaced by analysis.helix_analysis in 2.0.0.
+- analysis.hbonds.WaterBridgeAnalysis will be moved to analysis.hydrogenbonds.WaterBridgeAnalysis.
+
+
+The following parts of the readers/writers will be removed/changed in version 2.0.0:
+- Writer.write_next_timestep() is deprecated in favor of Writer.write().
+- Passing Timestep objects to Writer.write() is deprecated. In 2.0.0 only Universe or AtomGroup objects will be accepted.
+- The way in which the NCDFWriter handles scale factos will change in version 2.x (see Issue #2327 for more details).
+- When writing PDB files, MDAnalysis will no longer be using the last letter of the SegID to set the chainID in version 2.0.0.
+- The bfactors and tempfactors attributes (set by the PDB and MMTF parsers respectively), will be aliased in version 2.0.0.
+- When parsing TPR files, resids will be indexed from 1 rather than the current default of 0.
+
+The following part of the core and library components will be removed/changed in version 2.0.0:
+- lib.log.echo() is deprecated in favor of the new lig.log.ProgressBar.
+- core.universe.as_Universe() is deprecated.
+
+## Author statistics
+
+Altogether this represents the work of 42 contributors from around the world,
+and featured the work of 25 new contributors:
+Note: here is a list of all the new contributors' github handles, can someone please convert this to a list with their names and a link to their github profiles? i.e.:
+ - [Irfan Alibay][]
```

Comment:
Why not 
```
Irfan Alibay (@IAlibay)
```
or just the GH handles?

We can always link to AUTHORS.

Reply:
I was just imitating the 0.19.x blog post, whatever is easier is best here.